source/page/data.ts

```diff
@@ -5,105 +5,111 @@ import banner_2018_React from './2018-React/React-ChengDu.jpg';
+        },
+        {
+            title: t('coscon20_chengdu_branch_venue'),
+            date: '2020-10-24',
+            URL: 'https://fcc-cd.dev/activity/conference/coscon-2020-chengdu/',
+            banner: 'https://ows.blob.core.chinacloudapi.cn/files/assets/COS_Con_20_abaa9e2a66.png'
+        },
+        {
+            title: t('chengdu_web_full_stack_conference_2019'),
+            date: '2019-11-16',
+            URL: '2019/',
+            banner: banner_2019
+        },
+        {
+            title: t('chengdu_web_frontend_conference_2018'),
+            date: '2018-11-10',
+            URL: '2018/',
+            banner: banner_2018
+        },
+        {
+            title: t('chongqing_web_frontend_conference_2018'),
+            date: '2018-08-18',
+            URL: 'https://fcc-cd.dev/activity/conference/web-fe-conf-chongqing-2018/',
+            banner: 'http://wimg.huodongxing.com/logo/201807/9450206156400/123133704540930_v2.jpg@!wmlogo'
+        },
+        {
+            title: t('chengdu_react_tech_conference_2018'),
+            date: '2018-03-31',
+            URL: 'https://juejin.cn/post/6844903694845100040',
+            banner: banner_2018_React
+        },
+        {
+            title: t('xinyao_cup_code_for_city_hackathon'),
+            date: '2018-01-20',
+            URL: '2018/Code4City/',
+            banner: gallery_Code4City[0]
+        },
+        {
+            title: t('chengdu_web_frontend_conference_2017'),
+            date: '2017-11-19',
+            URL: '2017/',
+            banner: banner_2017
+        }
+    ].sort(({ date: A }, { date: B }) => +new Date(B) - +new Date(A));
 
-const { date: first } = summits.slice(-1)[0],
-    { date: last } = summits[0];
+const { date: first } = summits().slice(-1)[0],
+    { date: last } = summits()[0];
 
 const years = Math.ceil(+last.slice(0, 4) - +first.slice(0, 4)) + 1;
 
-export const features = [
+export const features = () => [
     {
-        title: '秉持 freeCodeCamp 国际社区精神',
-        summary:
-            'freeCodeCamp.org 全球最大的开源编程学习平台（GitHub 星标前十），旨在成为计算机教育界的维基百科',
+        title: t('adhering_to_freecodecamp_international_community_spirit'),
+        summary: t('freecodecamp_global_largest_open_source_learning_platform'),
         logo: 'https://github.com/freeCodeCamp.png'
     },
     {
-        title: '引领 freeCodeCamp 中文社区潮流',
-        summary:
-            '第一个制定日常活动方针，第一个设计城市社区 logo、吉祥物，第一个举办黑客马拉松、技术大会，第一个试验开源线下运营',
+        title: t('leading_freecodecamp_chinese_community_trend'),
+        summary: t('first_to_establish_daily_activities_guidelines'),
         logo: 'https://github.com/freeCodeCamp-China.png'
     },
     {
-        title: '成都最活跃的程序员社区',
-        summary: `${years} 年连办 ${summits.length} 场技术干货大会，合办活动、中小分享会、动手训练营、线上直播不计其数`,
+        title: t('chengdu_most_active_programmer_community'),
+        summary: t('years_continue_to_undertake', {
+            which_year: years.toString + ' ',
```

Comment:
`toString` 是一个方法（在对象上的一个函数），你要调用它而不是引用它：

```suggestion
            which_year: years.toString() + '',
```